### ISSUE TITLE
path clash

### ISSUE THREAD

**Issue by yuanpeterli** [nextcloud/cms_pico#114]
Is it possible to prepend the userid to a user's website path? Currently, if each user has an about-me webpage, the webpage path would clash. It is better to have user1 with path /user1/about-me, user2 with /user2/about-me, and so on. Is this possible to set up?

**Comment by PhrozenByte** [comment]
No, that's not possible right now I'm afraid. Since it's a good idea I've added it to #99, maybe we'll implement this with Pico CMS for Nextcloud v1.1..
